client/Invoice.js

```diff
@@ -1,18 +1,12 @@
-var defaultsort = {sort : {createdAt:1,total:1}};
-Session.set('sortInvoices', defaultsort);
-var defaultFilter = {};
-Session.set('filterInvoices', defaultFilter);
-
+//session values by default
+Session.set('invoiceSortOrder', 'asc');
+Session.set('invoiceSortBy', 'createdAt');
+Session.set('invoiceTimeRange', 'today');
 
 //returns data for template
 Template.Invoices.helpers({
-   	invoices: ()=> {
-   			cur = Invoices.find(Session.get('filterInvoices'),Session.get('sortInvoices'));
-            //console.log(cur.count());
-		return cur;
-	}
-  });
-
-Template.Invoices.onCreated(function () {
-  this.subscribe('invoices');
+    invoices: () => {
```

Comment:
if you pass the data down from iron router you don't have to setup a helper specifically. You can also pass something like this down from iron router:
```javascript
data: function() {
  return {
    invoices: Invoices.byTimeRange("all", this.params.query.sortedBy,this.params.query.sortOrder)
  };
}
```
then you can just access `invoices` directly in your template code